skops/io/_sklearn.py

```diff
@@ -13,6 +13,9 @@
     SquaredHinge,
     SquaredLoss,
 )
+from sklearn.metrics import DistanceMetric
+from sklearn.metrics._dist_metrics import METRIC_MAPPING
```

Comment:
wouldn't maybe a user be able to modify this and hence allow things which shouldn't be allowed?

Reply:
So would you suggest to hard-code the metrics instead?

Reply:
yeah, like I did for losses.